### DISCUSSION TITLE
How do I set the initial zoom level so that the whole graph is initially visible? (in 2.0.0+)

### DISCUSSION THREAD

**Discussion by OskarPersson** [patrykandpatrick/vico#552] [Questions] [answered]
I want to set the initial zoom level of my line chart such that the whole x-axis of the graph is visible

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can do this by turning off scrolling. (If you’d like detailed instructions, please let us know if you’re using Vico with Compose or views.) A limitation of this approach is described in #296. We plan on adding a dedicated parameter for the initial zoom factor, independent of scrolling.
